File: java to c++/DisconnectRequest.cpp
#include "DisconnectRequest.h"

namespace bgu
{
	namespace spl171
	{
		namespace net
		{
			namespace impl
			{
				namespace TFTP
				{
					namespace Commands
					{
						using Response = bgu::spl171::net::impl::TFTP::Commands::Responses::Response;

						short DisconnectRequest::getOpCode()
						{
							return OpCode::Disconnect::getValue();
						}
					}
				}
			}
		}
	}
}

File: Server/src/main/java/bgu/spl171/net/impl/TFTP/Commands/DisconnectRequest.java
package bgu.spl171.net.impl.TFTP.Commands;

import bgu.spl171.net.impl.TFTP.Commands.Responses.Response;

public class DisconnectRequest implements Command {

    @Override
    public short getOpCode() {
        return OpCode.Disconnect.getValue();
    }

}
